scripts/dev_deploy_tags.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "Key": "System",
+    "Value": "Authentication"
+  },
+  {
+    "Key": "Environment",
+    "Value": "dev"
```

Comment:
ideally, we'd tag with the actual environment name (ie. `dev|authdev{1,2}`)

Reply:
This file keeps the tagging between CLI deployment and deployment via secure pipeline in-sync
![Screenshot 2024-12-19 at 16 55 03](https://github.com/user-attachments/assets/ddf8e435-264b-4896-b6b2-7085146c11e5)
Actual Environment and SubEnvironment values are passed onto the template as parameters.